### PR TITLE
Style/responsiveness sort box

### LINKED ISSUE #32: Add responsiveness, styling to app
- [x] Add responsiveness for mobile/tablet/desktop

- [x] Rework Radio button styling to match design. [Potential resource](https://markheath.net/post/customize-radio-button-css)

- [x] We have fun space icons, see if there is any reason to use them (found in public>space)

### PR DESCRIPTION
**Pull request by alexthompson207**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [ ] Feature
- [ ] Refactor
- [x] Style
- [ ] Testing

### Are there any known bugs?

- [x] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Add responsive styling to all planets view and sort container. Added icons and removed radio button default styling.

### What ticket(s) do the changes resolve?

#32 
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
Only slightly tweaked the single planet view, didn't want to start adding responsive styling to this page without actual data being displayed, and didn't know what the data would look like. Also the responsive design needs a second opinion, and the breakpoints for media queries may need change.

### How to Test Changes?

Open site local, shrink and expand the homepage/all planets view. Site styles should change with screen size.

### Next Iterations

Add responsive styling to single planet view. Figure out proper text size for the entire website along with colors. Maybe refactor/add colors and/or icons to the sort icons

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -37,8 +37,8 @@ class App extends React.Component<{}, AllData> {
       <div className="App">
         <Header />
         <main>
-          <SortBox updateSort={this.updateSort} />
```
Comment (ConnorAndersonLarson) [inline src/components/App/App.tsx]: Should we revert this back to show everything?

File: src/components/Planetarium/planetarium.css [hunk 2]
```diff
@@ -2,6 +2,6 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: space-evenly;
-  margin: 3% 0;
```
Comment (ConnorAndersonLarson) [inline src/components/Planetarium/planetarium.css:5]: May want to switch margins to a rigid dimension (px?)

**Review by ConnorAndersonLarson — APPROVED** [review]
Just had some minor things that may want to be edited sooner rather than later, but looks good!

File: src/components/App/App.tsx (hunk 1 above)
Comment (alexthompson207) [reply src/components/App/App.tsx]: Yes good call, fixed!

File: src/components/Planetarium/planetarium.css (hunk 2 above)
Comment (alexthompson207) [reply src/components/Planetarium/planetarium.css:5]: Yeah sure, never really sure when to use px vs. %. I'll change it to px!

File: src/components/PlanetInfo/planetInfo.css [hunk 3]
```diff
@@ -4,11 +4,22 @@
...
+
 .planet-info-img {
   width: 200px;
   align-self: center;
 }
 
+.planet-info-box {
+  border: 5px solid #D0D7DF;
+  width: 70%;
+  align-self: center;
```
Comment (knees4bees) [inline src/components/PlanetInfo/planetInfo.css:19]: I think this `align-self: center` line might not be necessary given the setup of the flexbox it's in. (And similarly for the `align-self` line in the section above.)

**Review by knees4bees — APPROVED** [review]
Great work getting this styling done and adding the cool icons, @alexthompson207 !

File: src/components/PlanetInfo/planetInfo.css (hunk 3 above)
Comment (knees4bees) [reply src/components/PlanetInfo/planetInfo.css:19]: Okay, my bad, looks like this is needed after all!

**Pull request merged**